source/administration/plugins.rst

```diff
@@ -77,12 +77,163 @@ Plugins may have one or both of the following parts:
 
 Security
 --------
-Plugins are intentionally powerful and not artificially sandboxed in any way and effectively become part of the Mattermost server. Server plugins can execute arbitrary code alongside your server and webapp plugins can deploy arbitrary code in client browsers. 
+Plugins are intentionally powerful and not artificially sandboxed in any way and effectively become part of the Mattermost server. Server plugins can execute arbitrary code alongside your server and webapp plugins can deploy arbitrary code in client browsers.
 
 While this power enables deep customization and integration, it can be abused in the wrong hands. Plugins have full access to your server configuration and thus also to your Mattermost database. Plugins can read any message in any channel, or perform any action on behalf of any user in the webapp.
 
 You should only install custom plugins from sources you trust to avoid compromising the security of your installation.
 
+Plugin Signing
+---------------
+
+The Plugin Marketplace allows system administrators to download and install plugins from a central repository. Plugins installed via the Plugin Marketplace must be signed by a public key certificate trusted by the local Mattermost server.
+
+While the server ships with a default certificate used to verify plugins from the default Mattermost plugin marketplace, the server can be configured to trust different certificates and point at a different plugin marketplace. This document outlines the steps for generating a public key certificate and signing plugins for use with a custom plugin marketplace. It assumes access to the _`GNU Privacy Guard (GPG)]<https://gnupg.org>`_ tool.
```

Comment:
```suggestion
While the server ships with a default certificate used to verify plugins from the default Mattermost plugin marketplace, the server can be configured to trust different certificates and point at a different plugin marketplace. This document outlines the steps for generating a public key certificate and signing plugins for use with a custom plugin marketplace. It assumes access to the _`GNU Privacy Guard (GPG)<https://gnupg.org>`_ tool.
```

Reply:
Ah, that slipped past me. Thanks, @amyblais.